### PR TITLE
Fix up host protection docs failsafe port list etc

### PR DESCRIPTION
**Pull request by gunjan5**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by gunjan5** [comment]
the failsafe ports list was missing a bunch of ports from what felix allows be default

File: master/getting-started/bare-metal/bare-metal.md [hunk 1]
```diff
@@ -388,11 +388,14 @@ policy, {{site.prodname}} has a failsafe mechanism that keeps various pinholes o
 in the firewall.
 
 By default, {{site.prodname}} keeps port 22 inbound open on *all* host endpoints,
-which allows access to ssh; as well as inbound and outbound communications to
-ports 2379, 2380 (etcd's default ports), and to ports 6666, 6667 (etcd's ports
+which allows access to ssh; outbound UDP port 53 for DNS queries, 
+inbound UDP port 68 and outbound UDP 67 for DHCP; as well as inbound and outbound 
+communications to ports 179 ( Calico BGP peering ports),
```
Comment (bcreane) [inline master/getting-started/bare-metal/bare-metal.md]: extra space after opening '('

File: master/getting-started/bare-metal/bare-metal.md [hunk 2]
```diff
@@ -388,11 +388,14 @@ policy, {{site.prodname}} has a failsafe mechanism that keeps various pinholes o
 in the firewall.
 
 By default, {{site.prodname}} keeps port 22 inbound open on *all* host endpoints,
-which allows access to ssh; as well as inbound and outbound communications to
-ports 2379, 2380 (etcd's default ports), and to ports 6666, 6667 (etcd's ports
+which allows access to ssh; outbound UDP port 53 for DNS queries, 
+inbound UDP port 68 and outbound UDP 67 for DHCP; as well as inbound and outbound
```
Comment (bcreane) [inline master/getting-started/bare-metal/bare-metal.md]: this might be easier to digest in table format now that we have more than a couple of ports

Comment (gunjan5) [reply master/getting-started/bare-metal/bare-metal.md]: I think that's a great idea. I'll update it

**Comment by gunjan5** [comment]
@bcreane I've updated it to be in a table format. lmk what you think

File: master/getting-started/bare-metal/bare-metal.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ You can apply host endpoint policies to three types of traffic:
 same host.
 
 Set the `applyOnForward` flag to `true` to apply a policy to forwarded traffic.
-See [policy spec]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy#spec).
+See [GlobalNetworkPolicy spec]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy#spec).
```
Comment (doublek) [inline master/getting-started/bare-metal/bare-metal.md]: Shouldn't this link to `globalnetworkpolicy#spec` and not `networkpolicy#spec`.

File: v3.0/getting-started/bare-metal/bare-metal.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ You can apply host endpoint policies to three types of traffic:
 same host.
 
 Set the `applyOnForward` flag to `true` to apply a policy to forwarded traffic.
-See [policy spec]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy#spec).
+See [GlobalNetworkPolicy spec]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy#spec).
```
Comment (doublek) [inline v3.0/getting-started/bare-metal/bare-metal.md]: Shouldn't this link to globalnetworkpolicy#spec and not networkpolicy#spec.

File: v3.0/getting-started/bare-metal/bare-metal.md [hunk 5]
```diff
@@ -177,7 +177,7 @@ connectivity to a host:
     allowing traffic out to etcd's ports on any interface.
     
 -   Depending on your network, they may not cover all the ports that are
-    required; for example, your network may reply on allowing ICMP,
+    required; for example, your network may rely on allowing ICMP,
```
Comment (doublek) [inline v3.0/getting-started/bare-metal/bare-metal.md]: Nice eye. The same change perhaps needs to be done for `master` docs as well?

File: master/getting-started/bare-metal/bare-metal.md (hunk 3 above)
Comment (gunjan5) [reply master/getting-started/bare-metal/bare-metal.md]: ah good find! I'll fix it

**Comment by gunjan5** [comment]
ping @doublek @bcreane @emanic . I've made the changes you suggested, PTAL

File: master/getting-started/bare-metal/bare-metal.md [hunk 6]
```diff
@@ -387,12 +387,23 @@ To avoid completely cutting off a host via incorrect or malformed
 policy, {{site.prodname}} has a failsafe mechanism that keeps various pinholes open
 in the firewall.
 
-By default, {{site.prodname}} keeps port 22 inbound open on *all* host endpoints,
-which allows access to ssh; as well as inbound and outbound communications to
-ports 2379, 2380 (etcd's default ports), and to ports 6666, 6667 (etcd's ports
-as deployed by {{site.prodname}}'s self-hosted Kubernetes manifests).
+By default, {{site.prodname}} keeps the following ports open:
```
Comment (doublek) [inline master/getting-started/bare-metal/bare-metal.md]: Last nit, I think. The previous text emphasizes "all host endpoints". We should probably retain that piece of information as-is.

Comment (gunjan5) [reply master/getting-started/bare-metal/bare-metal.md]: that's a good point! I've added that line back, PTAL

**Comment by emanic** [comment]
Can you try to rebase off of the latest master and see if the `htmlproofer` error resolves?

**Comment by emanic** [comment]
thanks @gunjan5 !

**Pull request merged**